evm_loader/program/src/entrypoint.rs

```diff
@@ -395,11 +393,13 @@ fn do_finalize<'a>(program_id: &Pubkey, accounts: &'a [AccountInfo<'a>]) -> Prog
             code.to_vec()
         };
 
-        let executor_state = ExecutorState::new(ExecutorSubstate::new(), backend);
+        let gas_limit = u64::MAX;
+	    let config = evm::Config::default();
+        let executor_state = ExecutorState::new(ExecutorSubstate::new(gas_limit, config), backend);
```

Comment:
Do not create config there. Use `Config::default()` inside substate constructor

Reply:
Good point. Moved Config creation there, and the code became more concise and readable.